docs/monitoring.md

```diff
@@ -1038,6 +1038,22 @@ This example shows a list of Spark configuration parameters for a Graphite sink:
 "spark.metrics.conf.*.sink.graphite.prefix"="optional_prefix"
 "spark.metrics.conf.*.sink.graphite.regex"="optional_regex_to_send_matching_metrics"
 ```
+Also it could be done under spark session like:
+```
+import org.apache.spark.sql.SparkSession
```

Comment:
Why we need this example? You think it is difficult for users to know how to pass these parameter into Spark?